benchmarks/distributed/rpc/parameter_server/launcher.py

```diff
@@ -0,0 +1,320 @@
+            ),
+            rank=rank,
+            world_size=world_size,
+            rpc_backend_options=rpc_backend_options
+        )
+    else:
+        # trainers = [0, trainer_count)
+        if (USE_CUDA_RPC in trainer_configurations and
+            trainer_configurations[USE_CUDA_RPC] and
+            USE_CUDA_RPC in ps_configurations and
+            ps_configurations[USE_CUDA_RPC] and
+                configurations.parameter_server_count > 0):
+            ps_rank = get_parameter_server_rank(rank, configurations)
+            ps_name = get_name(
+                ps_rank,
+                configurations.trainer_count,
+                configurations.parameter_server_count
+            )
+            rpc_backend_options.set_device_map(
+                ps_name,
+                {rank: ps_rank}
+            )
+        trainer_name = get_name(
+            rank,
+            configurations.trainer_count,
+            configurations.parameter_server_count
+        )
+        rpc.init_rpc(
+            trainer_name,
+            rank=rank,
+            world_size=world_size,
+            rpc_backend_options=rpc_backend_options
+        )
+    rpc.shutdown()
+
+
+def get_json_config(file_name, id):
+    f = open(
+        os.path.join(
+            Path(__file__).parent, file_name
+        ),
+        "r"
+    )
+    return json.load(f)[id]
+
+
+def load_configurations(args):
+    benchmark_config_file = "configurations/benchmark_configurations.json"
+    benchmark_config = get_json_config(benchmark_config_file, args.bconfig_id)
+    return BenchmarkConfigurations(**benchmark_config)
+
+
+def get_data(data_class, data_config):
+    data_class = get_benchmark_data_map()[data_class]
+    return data_class(**data_config)
+
+
+def load_data(args):
+    data_config_file = "configurations/data_configurations.json"
+    data_config = get_json_config(data_config_file, args.dconfig_id)
+    return get_data(data_config["data_class"], data_config["configurations"])
+
+
+def get_model(model_class, model_config):
+    model_class = get_benchmark_model_map()[model_class]
+    return model_class(**model_config)
+
+
+def load_model(args):
+    model_config_file = "configurations/model_configurations.json"
+    model_config = get_json_config(model_config_file, args.mconfig_id)
+    return get_model(model_config["model_class"], model_config["configurations"])
+
+
+def load_trainer_configurations(args):
+    trainer_config_file = "configurations/trainer_configurations.json"
+    return get_json_config(trainer_config_file, args.tconfig_id)
+
+
+def load_parameter_server_configurations(args):
+    if args.pconfig_id == "None":
+        return None
+    ps_config_file = "configurations/parameter_server_configurations.json"
+    return get_json_config(ps_config_file, args.pconfig_id)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="RPC PS Benchmark")
+    parser.add_argument(
+        "--bconfig_id",
```

Comment:
after changing this to take a file path, let's also update the config names. We can call then --benchmark, --data, --model, --server, --trainer, for readability.